packages/cozy-viewer/src/Panel/getPanelBlocks.jsx

```diff
@@ -27,7 +27,11 @@ export const getPanelBlocksSpecs = (isPublic = false) => ({
     component: Qualification
   },
   konnector: {
-    condition: file => isFromKonnector(file) && !isPublic,
+    condition: file =>
+      isFromKonnector(file) &&
+      !isPublic &&
+      // When editing the doc from mespapiers, we don't want to display the konnector block
```

Comment:
nit: C'est "dès que le doc a été modifié par mes papiers, on n'affiche pas" ? Si oui, je trouve plus clair

// When doc has been edited from mespapiers, we don't want to display the konnector block